Amplify/Categories/Auth/AuthCategoryUserBehavior.swift

```diff
@@ -39,8 +39,21 @@ public protocol AuthCategoryUserBehavior: AnyObject {
     /// - Parameters:
     ///   - userAttributeKey: Attribute to be verified
     ///   - options: Parameters specific to plugin behavior
-    func resendConfirmationCode(forUserAttributeKey userAttributeKey: AuthUserAttributeKey,
-                                options: AuthAttributeResendConfirmationCodeRequest.Options?) async throws -> AuthCodeDeliveryDetails
+    @available(*, deprecated, renamed: "sendVerificationCode")
```

Comment:
Not entirely sure if this is needed, but would this be better? it would more accurately reflect the new method. I wonder if it provides any additional benefits such as click through (cmd click)?
```suggestion
    @available(*, deprecated, renamed: "sendVerificationCode(forUserAttributeKey:options:")
```

Reply:
sorry i missed a bracket, this should work: 

```suggestion
    @available(*, deprecated, renamed: "sendVerificationCode(forUserAttributeKey:options:)")
```